src/goSuggest.ts

```diff
@@ -183,6 +185,18 @@ export class GoCompletionItemProvider implements vscode.CompletionItemProvider {
 								item.insertText = new vscode.SnippetString(suggest.name + '(' + paramSnippets.join(', ') + ')');
 							}
 
+							if (wordAtPosition && wordAtPosition.start.character === 0 &&
+								suggest.class === 'type' && !goBuiltinTypes.has(suggest.name)) {
+								let prefix = 'func (' + suggest.name[0].toLowerCase() + ' *' + suggest.name + ')';
+								let auxItem = new vscode.CompletionItem(suggest.name + ' method', vscode.CompletionItemKind.Snippet);
+								auxItem.label = suggest.name + ' (new method)';
```

Comment:
What do you think of `'func (*' + suggest.name + ')` for `label` and `Method snippet` for `detail`?

Reply:
I experimented with you idea and found the the vscode engine does not show suggestions in some case where the `suggest.name` is not a prefix of the `label`. More specifically,  when completing an unexported type the suggestion is not shown.
For now I changed the `label` to be exactly `suggest.name` and kept your suggestion for the `label`.
Let me know what you think!

Reply:
You can get around that by adding `auxItem.filterText = suggest.name`

By default the `label` is used for displaying the item as well as filtering the item based on what is typed. If `filterText` is set, then that will be used for filtering instead

Reply:
Nice! Learned something new.
Done.